getGit.tex

```diff
@@ -160,6 +159,7 @@ \section{What, Why, and How?}
     	\item{Everyone uses git}
     		\begin{itemize}
     			\item{Major corporations}
+    			\item{The government}
```

Comment:
Sorry to nitpick, @Oddant1. "Governments" is more accurate here. US and state governments, but also governmental entities all over the world use git.

Reply:
No definitely nitpick. One of the biggest advantages of git here is it makes every single change extremely transparent. My buddy and I looked at installing git on his Mac and as expected it was just a very simple program to download and run. I included the name of the program in this latest commit